### ISSUE TITLE
DOC: Documentation refers to outdated 'postgres' protocol for sqlalchemy.

### ISSUE THREAD

**Issue by ingenieroariel** [geopandas/geopandas#2079]
Instances of

db_connection_url = "postgres://..."

should be replaced with

db_connection_url = "postgresql://

in several pages of the docs.

**Comment by martinfleis** [comment]
Thanks for the catch!

Just a question - aren't these two identical? Or is the `postgres` deprecated or something? I am trying to understand what makes it outdated (note that I am not using it much, so I don't follow the news on that front).

**Comment by ingenieroariel** [comment]
We found out the hard way today when it was just not accepted anymore. The specific error message was related to postgres protocol not supported.

**Comment by martinfleis** [comment]
Ahh, okay. Thanks!

PR changing `postgres` to `postgresql` is welcome!

**Comment by ingenieroariel** [comment]
```
diff --git a/doc/source/docs/user_guide/io.rst b/doc/source/docs/user_guide/io.rst
index 5d4ee9e..534891a 100644
--- a/doc/source/docs/user_guide/io.rst
+++ b/doc/source/docs/user_guide/io.rst
@@ -197,7 +197,7 @@ Spatial databases
 Writing to PostGIS::
 
     from sqlalchemy import create_engine
-    db_connection_url = "postgres://myusername:mypassword@myhost:5432/mydatabase";
+    db_connection_url = "postgresql://myusername:mypassword@myhost:5432/mydatabase";
     engine = create_engine(db_connection_url)
     countries_gdf.to_postgis("countries_table", con=engine)
 
diff --git a/geopandas/geodataframe.py b/geopandas/geodataframe.py
index e513ee6..1bcc6ce 100644
--- a/geopandas/geodataframe.py
+++ b/geopandas/geodataframe.py
@@ -654,7 +654,7 @@ class GeoDataFrame(GeoPandasBase, DataFrame):
         PostGIS
 
         >>> from sqlalchemy import create_engine  # doctest: +SKIP
-        >>> db_connection_url = "postgres://myusername:mypassword@myhost:5432/mydb"
+        >>> db_connection_url = "postgresql://myusername:mypassword@myhost:5432/mydb"
         >>> con = create_engine(db_connection_url)  # doctest: +SKIP
         >>> sql = "SELECT geom, highway FROM roads"
         >>> df = geopandas.GeoDataFrame.from_postgis(sql, con)  # doctest: +SKIP
@@ -1702,7 +1702,7 @@ box': (2.0, 1.0, 2.0, 1.0)}], 'bbox': (1.0, 1.0, 2.0, 2.0)}
         --------
 
         >>> from sqlalchemy import create_engine
-        >>> engine = create_engine("postgres://myusername:mypassword@myhost:5432\
+        >>> engine = create_engine("postgresql://myusername:mypassword@myhost:5432\
 /mydatabase")  # doctest: +SKIP
         >>> gdf.to_postgis("my_table", engine)  # doctest: +SKIP
 
diff --git a/geopandas/io/sql.py b/geopandas/io/sql.py
index 4aa3767..485cf93 100644
--- a/geopandas/io/sql.py
+++ b/geopandas/io/sql.py
@@ -138,7 +138,7 @@ def _read_postgis(
     PostGIS
 
     >>> from sqlalchemy import create_engine  # doctest: +SKIP
-    >>> db_connection_url = "postgres://myusername:mypassword@myhost:5432/mydatabase"
+    >>> db_connection_url = "postgresql://myusername:mypassword@myhost:5432/mydatabase"
     >>> con = create_engine(db_connection_url)  # doctest: +SKIP
     >>> sql = "SELECT geom, highway FROM roads"
     >>> df = geopandas.read_postgis(sql, con)  # doctest: +SKIP
@@ -362,7 +362,7 @@ def _write_postgis(
     --------
 
     >>> from sqlalchemy import create_engine  # doctest: +SKIP
-    >>> engine = create_engine("postgres://myusername:mypassword@myhost:5432\
+    >>> engine = create_engine("postgresql://myusername:mypassword@myhost:5432\
 /mydatabase";)  # doctest: +SKIP
     >>> gdf.to_postgis("my_table", engine)  # doctest: +SKIP
     """
```

**Comment by ingenieroariel** [comment]
https://github.com/geopandas/geopandas/pull/2082

**Comment by martinfleis** [comment]
Closed by #2082
